react/FilePicker/FilePickerBreadcrumb.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useCallback } from 'react'
+import PropTypes from 'prop-types'
+
+import Typography from '../Typography'
+import Icon from '../Icon'
+import RightIcon from '../Icons/Right'
+import BottomIcon from '../Icons/Bottom'
+import useBreakpoints from '../hooks/useBreakpoints'
+
+const previousColorStyle = { color: 'var(--actionColorActive)' }
+
+const FilePickerBreadcrumb = ({ path, onBreadcrumbClick }) => {
+  const { isMobile } = useBreakpoints()
+  const hasPath = path && path.length > 0
+
+  const navigateTo = useCallback(folder => () => onBreadcrumbClick(folder), [
+    onBreadcrumbClick
+  ])
+
+  return (
+    <Typography variant="h4" className={'u-flex u-flex-items-center'}>
+      {isMobile
+        ? hasPath && path[path.length - 1].name
+        : hasPath &&
+          path.map((folder, index) => {
+            if (index < path.length - 1) {
+              return (
+                <Fragment key={index}>
+                  <span
+                    className={'u-c-pointer'}
+                    style={previousColorStyle}
+                    onClick={navigateTo(folder)}
+                  >
+                    {folder.name}
+                  </span>
+                  {!isMobile && (
+                    <Icon icon={RightIcon} style={previousColorStyle} />
+                  )}
+                </Fragment>
+              )
+            } else {
+              return (
+                <span key={index}>
+                  {folder.name}
+                  {path.length >= 2 && isMobile && <Icon icon={BottomIcon} />}
```

Comment:
`isMobile` seems to be always `false` here due to the ternary check on line 22. Isn't it?